site/src/components/Navbar.tsx

```diff
@@ -158,6 +165,10 @@ export const Navbar: VFC<NavbarProps> = ({
     }
   }, [isDesktopSize, displayMobileNav]);
 
+  const closeDrawer = useCallback(() => {
```

Comment:
Creates and passes down a function for closing the sidebar. This is used to close the sidebar when the user taps on a search result.

Reply:
I don't think you need this, as `onClose` already exists and does the same thing - you could pass that to `Search` from `MobileNavItems` instead.